src/cellfinder_core/detect/filters/volume/volume_filter.py

```diff
@@ -146,10 +147,11 @@ def get_results(self) -> List[Cell]:
             cell_volume = len(cell_points)
 
             if cell_volume < max_cell_volume:
-                cell_centre = get_structure_centre_wrapper(cell_points)
+                cell_centre = get_structure_centre(cell_points)
+                cell_centre_point = Point(cell_centre[0], cell_centre[1], cell_centre[2])
```

Comment:
Not sure why a point is being constructed here, but not in the lines changed below? I'd say don't bother using `Point` here.